api/envoy/admin/v2alpha/config_dump.proto

```diff
@@ -70,9 +69,52 @@ message ListenersConfigDump {
     api.v2.Listener listener = 2;
 
     // The timestamp when the Listener was last updated.
+    // FIXME: reviewer we may eventually want last updated and last _successfully_ updated.
+    // Given similarity between last_updated (attempt or success) and
+    // last_update should this maybe be last_update_attempt so last_update
```

Comment:
Yeah, I would make this `last_update_success` and `last_update_attempt` explicitly now.